modules/150-user-authn/hooks/get_dex_client_crds_test.go

```diff
@@ -93,7 +97,13 @@ spec:
   },
   "legacyID": "dex-client-opendistro:test",
   "legacyEncodedID": "mrsxqlldnruwk3tufvxxazlomruxg5dsn45hizltotf7fhheqqrcgji",
-  "clientSecret": "test"
+  "clientSecret": "test",
+  "labels": {
+    "test-label": "test-value"
+  },
+  "annotations": {
+    "test-annotation": "test-value"
```

Comment:
Here you can see that new fields appear in secret, but we do not check how exceptions will be handled. I checked locally, no exceptions appear, it's great. But you need to reflect this in tests so that you don't encounter a situation where this check will be needed in the future.